appcli/launcher_cli.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# # -*- coding: utf-8 -*-
+
+"""
+The launcher command available when running the CLI.
+
+Responsible for creating launcher commands.
+________________________________________________________________________________
+
+Created by brightSPARK Labs
+www.brightsparklabs.com
+"""
+
+# standard library
+import os
+
+# vendor libraries
+import click
+
+# internal libraries
+from .models import CliContext, Configuration
+
+# ------------------------------------------------------------------------------
+# CLASSES
+# ------------------------------------------------------------------------------
+
+
+class LauncherCli:
+
+    # ------------------------------------------------------------------------------
+    # CONSTRUCTOR
+    # ------------------------------------------------------------------------------
+
+    def __init__(self, configuration: Configuration):
+
+        self.configuration: Configuration = configuration
+
+        @click.command(help="Outputs an appropriate launcher bash script to stdout")
+        @click.pass_context
+        def launcher(ctx):
+
+            cli_context: CliContext = ctx.obj
+            APP_VERSION = os.environ.get("APP_VERSION")
+
+            # Variables are defaulted in the script so that they can be overridden if desired
+            print(
+                f"""
+#!/bin/bash
+
+docker run \\
+    --rm \\
+    --volume /var/run/docker.sock:/var/run/docker.sock \\
+    {self.configuration.docker_image}:{APP_VERSION} \\
+        --configuration-dir "${{TELPASS_CONFIG_DIR:-{cli_context.configuration_dir}}}" \\
```

Comment:
You can't hardcode TELPASS in here. It changes for whichever client is using the lib. I.e. you need to pull app name out of the `configuration`

Reply:
Oh yep my bad, missed that one. Thanks!